_posts/0000-01-02-DanSam5K.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Okay I am excited you got to review and provided feedback as to what next step is required of me.